File: HANSOLO.php
$file = fopen("HANSOLO.hs", "w"); // open file for writing

for ($i = 0; $i < 1000000; $i++) {
    fwrite($file, "HANSOLO\n"); // write "HANSOLO" to file followed by a newline character
}

fclose($file); // close file

File: HANSOLO.dart
import 'dart:io';

void main() {
  final hansolo = 'HANSOLO';
  final file = File('HANSOLO.hs');
  final sink = file.openWrite();

  for (var i = 0; i < 1000000; i++) {
    sink.write('$hansolo\n');
  }

  sink.close();
}
